### DISCUSSION TITLE
希望可以增加自定义自动重启等待时间的功能

### DISCUSSION THREAD

**Discussion by Ashe428** [LmeSzinc/AzurLaneAutoScript#1829] [Ideas]
我的alas以及模拟器都运行在nas的虚拟机中。由于没有独立显卡（集成显卡直通失败，只能把显卡虚拟化再直通），再加上多重嵌套虚拟的缘故，模拟器性能很差，每次重启游戏都会在登录界面卡一阵子，导致alas多次运行“重启设置”后触发人工接管。希望可以增加自定义触发自动重启的等待时间的功能，这样我就不用每天12点都手动开一遍alas了。

ps：今天在群里问过这个问题，群主给的建议是直接在linux中开模拟器。但由于我的nas用的是unraid os，直接配置模拟器并不是很方便，且我担心会对本来bug就多的unraid产生影响，最终还是放弃这一方案。

最后感谢开发人员一直以来对alas的维护以及功能的完善，好人一生平安！

**Comment by LmeSzinc** [comment]
录个屏来看看，录屏性能有问题拿手机拍屏幕就可以了。包含这一整个过程：
1. 同时拍摄到alas的log和游戏画面
2. 在未启动游戏的情况下直接运行alas调度器，让alas启动游戏
3. 等待alas超时的时候手动停止alas
4. 手动点击游戏直到进入游戏主界面
